_sass/custom.scss

```diff
@@ -131,7 +131,7 @@ h2:not(:first-child) {
   display: none;
   &.is-open {
     display: block;
-    height: 100%;
+    height: 85%;
```

Comment:
I don't think this is the right place to change height - that's the background which should always be at 100%

Either try tweaking the `modal-content` class OR just add some padding/margins to the elements in the modal (which actually gives things a bit more room to breathe)